modules/exploits/linux/http/logsign_exec.rb

```diff
@@ -0,0 +1,77 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => 'Logsign Remote Command Injection',
+      'Description'    => %q{
+        This module exploits an command injection vulnerability in Logsign.
+        By exploiting this vulnerability, unauthenticated users can execute
+        arbitrary code under the root user.
+
+        Logsign has a publicly accessible endpoint. That endpoint takes a user
+        input and then use it during operating system command execution without
+        proper validation.
+
+        This module was tested against 4.4.2 and 4.4.137 versions.
+      },
+      'License'         => MSF_LICENSE,
+      'Author'          =>
+        [
+          'Mehmet Ince <mehmet@mehmetince.net>'  # author & msf module
```

Comment:
You have been killing it with these disclosures, @mmetince. I'm very happy that you're publishing these modules!